tests/src/test/scala/actionContainers/Ruby25ActionContainerTests.scala

```diff
@@ -0,0 +1,408 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This will also go with #3467.

Reply:
Thanks. I will remove this once #3467 get merged.

Reply:
I have removed this at 3cc287c1f67181c1d6c66ff59bbda01b913bcfa1